src/tests/function/ltrim_rtrim.rs

```diff
@@ -0,0 +1,83 @@
+use crate::*;
+
+test_case!(ltrim_rtrim, async move {
+    use Value::Str;
+    let test_cases = vec![
+        ("CREATE TABLE Item (name TEXT)", Ok(Payload::Create)),
+        (
+            r#"INSERT INTO Item VALUES (" zzzytest"), ("testxxzx ")"#,
+            Ok(Payload::Insert(2)),
+        ),
+        (
+            r#"SELECT Ltrim(name) AS test FROM Item"#,
+            Ok(select!(
+                "test"
+                Str;
+                "zzzytest".to_owned();
+                "testxxzx ".to_owned()
+            )),
+        ),
+        (
+            r#"SELECT Ltrim(name, ' xyz') AS test FROM Item"#,
+            Ok(select!(
+                "test"
+                Str;
+                "test".to_owned();
+                "testxxzx ".to_owned()
+            )),
+        ),
+        (
+            r#"SELECT Rtrim(name) AS test FROM Item"#,
+            Ok(select!(
+                "test"
+                Str;
+                " zzzytest".to_owned();
+                "testxxzx".to_owned()
+            )),
+        ),
+        (
+            r#"SELECT Rtrim(name, 'xyz ') AS test FROM Item"#,
```

Comment:
You are currently still working on it, so I'll simply leave as a comment.
As other SQL keywords are used as capital words (`SELECT`, `AS`, `FROM`...), it would be also good to use `LTRIM` and `RTRIM`.

Reply:
Thank you for tidying up.
I may still be confused with asf function name 😰